config/manager/manager.yaml

```diff
@@ -34,6 +34,7 @@ spec:
                   fieldPath: metadata.namespace
           securityContext:
             allowPrivilegeEscalation: false
+            readOnlyRootFilesystem: true
```

Comment:
This doesn't cover kube rbac proxy container.
Let's set this in podSpec.securityContext layer ? 

https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.26/#podspec-v1-core

Reply:
This option is not available for the podSpec.securityContext, it can only be set per container.

I indeed missed the kube-rbac-proxy, will add that one too.

Reply:
:+1: ack.